proto/kvrpcpb.proto

```diff
@@ -401,6 +322,24 @@ message GCResponse {
     KeyError error = 2;
 }
 
+// Delete a range of data from TiKV. Similar to a scan, but deleting a range, rather than reading.
+message DeleteRangeRequest {
```

Comment:
@MyonKeminta PTAL. I remember this command is not recommended for use.

Reply:
You are right. This is not recommended to use, which I think should be mentioned in the comment. By the way, it's not similar to scan, since it may invoke RocksDB's `delete_files_in_range` or `delete_range` to delete the data, and it will not consider constraints of transaction.